### ISSUE TITLE
Table contains more than 32 columns

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#51]
I've tried to build the latest master on dev, but unfortunately I got errors like:

```
error: Table contains more than 32 columns. Consider enabling the `64-column-tables` feature to enable diesels support for tables with more than 32 columns.
error[E0433]: failed to resolve: use of undeclared crate or module `plant_detail`
```

@aidnurs @buenaflor Do you have any idea on what I could have done wrong?

`backend/README.md` states the following commands:

```
cargo install diesel_cli --no-default-features --features postgres
LC_ALL=C ~/.cargo/bin/diesel setup # needs db
LC_ALL=C ~/.cargo/bin/diesel migration run # needs db
cargo install typeshare-cli
cargo build
cargo run
```

_Originally posted by @lukashartl in https://github.com/ElektraInitiative/PermaplanT/issues/37#issuecomment-1433926689_

**Comment by markus2330** [comment]
Thank you for reporting this problem!

**Comment by markus2330** [comment]
Should be fixed in #52.
